src/js/admin.js

```diff
@@ -8,6 +8,7 @@
 /* global _ */
 /* global ace */
 /* global console */
+/* global semver */
```

Comment:
@Zefau Just out of curiosity: This is the frontend, isn't it? How is `semver` loaded there?

Reply:
Don't know how that works, but I tested it successfully. Maybe you could adapt the change and verify?

Reply:
Ok, can this now be released as a version or not?